core/hybrisadaptor.cpp

```diff
@@ -453,6 +443,7 @@ void HybrisAdaptor::stopSensor()
         entry->setIsRunning(false);
         running_ = false;
         shouldBeRunning_ = false;
+        inStandbyMode_ = false;
```

Comment:
Does this mean stop() makes implicit setStandbyOverride(false) too? Which would sound like API change to me.